PyRDF/__init__.py

```diff
@@ -55,3 +55,6 @@ def include(includes_list):
         includes_list = [includes_list]
 
     includes.extend(includes_list)
+
+def broadcastInitialization(fun, *args, **kwargs):
+    current_backend.registerInitialization(fun, *args, **kwargs)
```

Comment:
I would not mix snake and camel case. I think the library is using snake case.

Reply:
Indeed, snake case is what [PEP8 recommends](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names).